packages/admin-panel/src/table/ExpansionContainer.js

```diff
@@ -24,14 +24,50 @@ export const ExpansionContainer = styled.div`
   .rt-tr {
     padding-right: 5px;
   }
+`;
 
-  &:after {
```

Comment:
fix a z-index front end bug where the box shadow was blocking clicking of components under it